open_lm/utils/transformers/config.py

```diff
@@ -13,18 +13,20 @@ def __init__(
         vocab_size: int = 50304,
         post_embed_norm: bool = False,
         weight_tying: bool = False,
+        model_norm: str = "default_layer_norm",
```

Comment:
can we delete this file and just use `open_lm/config.py`?

Reply:
also instead of the path being `open_lm/utils/transformers..` for this code, would it make sense to rename the dir `open_lm/generation` or something like that?

Reply:
> can we delete this file and just use open_lm/config.py?

Can we do this in a separate PR? open_lm/config.py is also used for eval, so this PR would then need to modify (+test) the eval code too. 

> would it make sense to rename the dir open_lm/generation or something like that?
I think since we use this for eval, probably better to leave the name for now?